common/diff.cpp

```diff
@@ -1,7 +1,5 @@
 #include "diff.h"
-#include "../lib/dtl/dtl/dtl.hpp"
-
-#include <algorithm>
+#include <dtl/dtl.hpp>
```

Comment:
This fails in CI, maybe we should keep it like previously and fix flake to support it?

Reply:
I've fixed it in https://github.com/tea-io/tea/pull/153. Should work after merging it and a rebase.

Reply:
I am not sure if this is a good idea. I made a mistake and removed bad lines (false positives from lsp), but our submodule for dtl was something simply. Anyone can build our software from source without `nix`. Of course, this option is more dangerous due to different versions of protobuf, but it is still available. Users can still build from source by downloading the library themselves, but this change makes it a bit more complicated. This flake does exactly the same thing as the submodule. So I stay with `submodule` but if you want new flake changes, I think we should add in `README.md` how to satisfy this dependency itself without nix. @jagtym

Reply:
Okay, I will merge your fix. We will add more information about this in the docs.